functions-and-operators/json-functions.md

```diff
@@ -56,17 +57,32 @@ TiDB supports most of the JSON functions that shipped with the GA release of MyS
 | [JSON_DEPTH(json_doc)][json_depth] | Returns the maximum depth of a JSON document |
 | [JSON_LENGTH(json_doc[, path])][json_length] | Returns the length of a JSON document, or, if a path argument is given, the length of the value within the path |
 | [JSON_TYPE(json_val)][json_type] | Returns a string indicating the type of a JSON value |
-| [JSON_VALID(json_val)][json_valid] | Returns 0 or 1 to indicate whether a value is valid JSON |
+| [JSON_VALID(json_doc)][json_valid] | Checks if a json_doc is valid JSON. Useful for checking a column before converting it to the json type. |
+
+## Utility Functions
+
+| Function Name and Syntactic Sugar | Description |
+| --------------------------------- | ----------- |
```

Comment:
Do we use this "Syntactic Sugar" label elsewhere? It looks like other pages just label this column "Name". The prototype of a function isn't what I think of when I think of "syntactic sugar".

Reply:
It comes from here: https://mysqlserverteam.com/inline-json-path-expressions-in-mysql-5-7/ -- I think "Function Name and Aliases" might be easier to read. Would you agree?